profile/toplev_metrics.json

```diff
@@ -0,0 +1,107 @@
+{"Level_1": 
```

Comment:
if this file is copied from the pmu tools repo, it should be fetched at the setup time and should not be in this repo.

Reply:
This file is not from pmu tools repo. JSON structure would be easier to parse and check the relations between nodes.